src/main/java/task/Task.java

```diff
@@ -0,0 +1,45 @@
+package task;
+
+public class Task {
+    protected String description;
+    protected boolean isDone;
+
+    public Task(String description) {
+        this.description = description;
+        this.isDone = false;
+    }
+
+    public String getStatusIcon() {
+        if (isDone) {
+            return "\u2713";
+        } else {
+            return "\u2718";
+        }
+    }
+
+    public String getTypeOfTask() {
+        return "NIL";
+    }
+
+    public String getFormattedDescription(){
+        if (description.contains("/")) {
+            String[] slicedDescription = description.split("/", 2);
+            String what = slicedDescription[0];
+            String when = slicedDescription[1].trim();
+            String[] slicedWhen = when.split(" ", 2);
+            when = slicedWhen[0] + ": " + slicedWhen[1];
+            return what + "(" + when + ")";
```

Comment:
Should the variables be named "taskDescription" and "taskTime" rather than "what" and "when"?